src/pages/nutrition/productCard.jsx

```diff
@@ -0,0 +1,109 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import Zoom from "react-medium-image-zoom";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import offService from "../../off";
+
+// import { basicNutriments } from "./nutritionFields";
+
+export default function ProductNutriments({ setNutriments, nutriments }) {
+  const [page, setPage] = React.useState(1);
+  const [index, setIndex] = React.useState(0);
+  const [products, setProducts] = React.useState([]);
+
+  const product = products[index];
+  const productName = product
+    ? product.product_name
+      ? product.product_name
+      : "Brand Unknown"
+    : "Loading...";
+
+  React.useEffect(() => {
+    const productListUrl = offService.getNutritionToFillUrl({ page });
+
+    fetch(productListUrl)
```

Comment:
nit: maybe just for consistency you could replace the `fetch` with `axios` (something like `axios.get( ... )`)

Reply:
Okay. It is gonna take time because I'm not familiar with axios yet

Reply:
hey @Shmigolk , please  ignore my comment and leave it as is